src/containers/pages/FocusInvestigation/map/planCompletion/index.tsx

```diff
@@ -87,11 +90,11 @@ export class PlanCompletion extends React.Component<
           })
           .catch(e => {
             // catch the error if one is returned from OpenSRP
-            alert(PLAN_STATUS_UPDATE_ERROR);
+            growl(`{PLAN_STATUS_UPDATE_ERROR}: ${e.message}`, { type: toast.TYPE.ERROR });
```

Comment:
@p-netm how can we test these grown notifications?

Reply:
investigating this,

 currently i don't have a good way of observing how they behave from within a test. 

What i tried doing was using `querySelector` on the document.
While this might capture the toast, the toast does not seem to behave as expected in terms of auto closing, The library itself uses redux action to force the toast to auto close,
I tried using jest timers but those did not help,
I also tried simulating clicks on the toast's close button from within the test , but this only appended css classes for the auto close transitions, but the toast will still be there on follow up `document.querySelector`.

Reply:
Is it possible to mock the growl function and test it that way?

Reply:
this i can do